CONTRIBUTING.md

```diff
@@ -42,32 +48,43 @@ The following rule governs documentation contributions:
+
+Creating branches within the project repositories itself requires write access to the repository and will done by code owners for specific purposes. For contribution into the branches of the project the "fork and pull model" is used. You need to create the branch for your contribution within a fork of the repository (see [part three of "GitHub for Poets"](https://youtu.be/_NrSWLQsDL4) for an friendly explanation or ["Working with forks"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks).
+
+## Creating Pull Requests
+
+For adding modifications to a branch in CAMARA, a pull request has to be initiated. When you open a pull request, you're proposing your changes and requesting that someone reviews and pulls in your contribution and merges them into their branch. Pull requests show diffs, or differences, of the content from both branches. The changes, additions, and subtractions are shown in different colors.
+
+Anyone with read access to a repository can create a pull request. You can specify which branch you'd like to merge your changes into when you create your pull request. You can link a pull request to an issue to show that a fix is in progress and to automatically close the issue when someone merges the pull request (see ["Linking a pull request to an issue"](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)).
+
+By using GitHub's @mention feature in your pull request message, you can ask for feedback from specific people.
+
+For more information on pull requests watch the third episode of ["GitHub for Poets"](https://www.youtube.com/watch?v=_NrSWLQsDL4). For a deep dive, see GitHub's docs:
+*	[About pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+*	[Propose changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests)
+*	[Collaborate with pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests)
+
+We kindly ask you to follow the [Pull Request Checklist](#Pull-Request-Checklist) to ensure reviews can happen accordingly.
 
 ## Pull Request Checklist
 
+* Work on a fork of the repository.
+
 * Branch from the main branch and, if needed, rebase to the current main branch before submitting your pull request. If it doesn't merge cleanly with main you may be asked to rebase your changes.
```

Comment:
Suggest add: "(see this [Git rebase tutorial for help](https://www.codementor.io/git/tutorial/git-rebase-split-old-commit-master)"

Reply:
Is added now

Reply:
@Kevsy @MarkusKuemmerle My impression is that the "Git rebase tutorial" above is about a different specific use case of rebase (= rewriting the history of a branch, e.g. to spilt commits). I would recommend instead to point to the GitHub manual page [Keeping your pull request in sync with the base branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/keeping-your-pull-request-in-sync-with-the-base-branch) and maybe in addition [More ways to keep your pull request branch up-to-date](https://github.blog/changelog/2022-02-03-more-ways-to-keep-your-pull-request-branch-up-to-date/).

Reply:
New links are included